GalerkinToolkitExamples/src/poisson.jl

```diff
@@ -263,90 +298,120 @@ function main_hand_written(params)
     Tgrad = SVector{D,T}
     Tx = Tgrad
     TJ = SMatrix{D,D,T}
```

Comment:
Replace `SMatrix{D,D,T}` with `typeof(zero(Tx)*transpose(zero(Tx)))`. The first one is not a concrete type (my bad).

Reply:
Thanks! done!